web/src/components/tasks/business-formation/DbaFormationPaginator.tsx

```diff
@@ -0,0 +1,215 @@
+import { Content } from "@/components/Content";
+import { ModalTwoButton } from "@/components/ModalTwoButton";
+import { Button } from "@/components/njwds-extended/Button";
+import { HorizontalStepper } from "@/components/njwds-extended/HorizontalStepper";
+import { TaskCTA } from "@/components/TaskCTA";
+import { DbaFormationSteps } from "@/components/tasks/business-formation/DbaFormationSteps";
+import { DbaFormationStepsConfiguration } from "@/components/tasks/business-formation/DbaFormationStepsConfiguration";
+import { AuthAlertContext } from "@/contexts/authAlertContext";
+import { BusinessFormationContext } from "@/contexts/businessFormationContext";
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { MediaQueries } from "@/lib/PageSizes";
+import analytics from "@/lib/utils/analytics";
+import { scrollToTopOfElement } from "@/lib/utils/helpers";
+import { FormationFormData } from "@businessnjgovnavigator/shared/formationData";
+import { emptyProfileData } from "@businessnjgovnavigator/shared/profileData";
+import { useMediaQuery } from "@mui/material";
+import { ReactElement, ReactNode, useContext, useEffect, useRef, useState } from "react";
+
+export const DbaFormationPaginator = (): ReactElement => {
+  const { userData } = useUserData();
+  const { state, setStepIndex } = useContext(BusinessFormationContext);
+  const { isAuthenticated, setRegistrationModalIsVisible } = useContext(AuthAlertContext);
+  const { Config } = useConfig();
+
+  const stepperRef = useRef<HTMLDivElement>(null);
+  const isMounted = useRef(false);
+  const isDesktop = useMediaQuery(MediaQueries.desktopAndUp);
+  const [showCtaModal, setShowCtaModal] = useState<boolean>(false);
+
+  const [stepperState, setStepperState] = useState(
+    DbaFormationStepsConfiguration.map((value, index) => {
+      return {
+        name: value.name,
+        hasError: false,
+        isComplete: index == 0,
+      };
+    })
+  );
+
+  useEffect(() => {
+    if (isMounted.current) {
+      scrollToTopOfElement(stepperRef.current, { isDesktop });
+    }
+  }, [state.stepIndex, isDesktop]);
+
+  const moveToStep = (stepIndex: number): void => {
+    setStepIndex(stepIndex);
+  };
+
+  const onPreviousButtonClick = (): void => {
+    setStepperState((stepperState) => {
+      stepperState[1].isComplete = state.stepIndex != 2;
+      return stepperState;
+    });
+    moveToStep(state.stepIndex - 1);
+  };
+
+  const isNotDba =
```

Comment:
it took me a solid moment to grasp why we would be checking if it "isNotDba" in the "DbaFormationPaginator"... I think the component should be renamed to "Nexus" or something, right?